index.js

```diff
@@ -65,6 +66,15 @@ const runInterval = async(docker, options) => {
 };
 
 module.exports.start = (options) => {
+  log = Logr.createLogger({
```

Comment:
any reason to not configure this on top of file?

Reply:
Later we might want to add CLI options that affect how the logger is configured, but I'll go ahead and move it to the top for now if there's no plans for that.